README.md

```diff
@@ -8,8 +8,8 @@ Sometimes your rows might come in a nested format or it might have a representat
 import * as resolve from 'table-resolver';
 
 // Or you can cherry-pick
-import { index } from 'table-resolver';
-import { index as resolveIndex } from 'table-resolver';
+import { nested } from 'table-resolver';
+import { nested as resolveNested } from 'table-resolver';
 ```
 
 ## API
```

Comment:
How about:
```javascript
function resolve({ columns, method }, options = { indexKey: "_index" }) {
...
ret = {
  [indexKey]: rowIndex,
  ...result
  ...rowData,
 ...ret,
};
...
```
That way the default is unchanged, users can overwrite it, and if they want to overwrite it *and* still have the index available then they can change what it's called.